torch/distributed/_tensor/op_schema.py

```diff
@@ -209,21 +212,26 @@ def __repr__(self) -> str:
 
     def __str__(self) -> str:
         args_sharding: List[str] = []
-        mesh = None
+        mesh_shape = None
         for arg in self.args_schema:
             if isinstance(arg, DTensorSpec):
                 args_sharding.append(str(arg))
-                mesh = arg.mesh
+                mesh_shape = tuple(arg.mesh.mesh.shape)
```

Comment:
suggestion: since we already have `device_mesh.ndim` and `device_mesh.size`, shall we add `device_mesh.shape` to avoid this `mesh.mesh.shape`?

Reply:
We can do that yeah, I'll leave it as a follow up PR